{
  "name": "CVE-2021-4146",
  "seq": "2021-4146",
  "type": "CAN",
  "status": "Candidate",
  "phase": {
    "text": "Assigned",
    "date": "20211221"
  },
  "desc": "Business Logic Errors in GitHub repository pimcore/pimcore prior to 10.2.6.",
  "refs": [
    {
      "text": "https://huntr.dev/bounties/47b37054-cafe-4f48-8b40-c86efc7fb760",
      "source": "CONFIRM",
      "url": "https://huntr.dev/bounties/47b37054-cafe-4f48-8b40-c86efc7fb760"
    },
    {
      "text": "https://github.com/pimcore/pimcore/commit/7011922f7f0f97a82d8c378559b91fcdb34604a6",
      "source": "MISC",
      "url": "https://github.com/pimcore/pimcore/commit/7011922f7f0f97a82d8c378559b91fcdb34604a6"
    }
  ],
  "votes": {}
}
